tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -66,7 +66,7 @@ private HttpRequest generateRequest(final BufferedReader bufferedReader) throws
         RequestLine requestLine = RequestLine.of(bufferedReader.readLine());
         RequestHeaders requestHeaders = generateRequestHeaders(bufferedReader);
         RequestBody requestBody = null;
-        if (requestLine.getMethod().equalsIgnoreCase("POST")) {
+        if (requestHeaders.getContentLength() > 0) {
```

Comment:
값을 꺼내서 처리하지 말고 객체에게 물어보세요.

```suggestion
        if (requestHeaders.hasContentLength()) {
```